### ISSUE TITLE
Hyperparameters Release

### ISSUE THREAD

**Issue by katie-cathy-hunt** [microsoft/DialoGPT#22]
Can you guys share the hyperparameters of different model sizes i.e. small, medium, and large?

https://github.com/microsoft/DialoGPT/blob/75a4197188a1addf22c5eaea23f16d3b598635d7/LSP_train.py#L46-L82

**Comment by dreasysnail** [comment]
I can share with you some experiences from us. For most of the experiments, we use learning rate ranging from 1e-4 to 1e-5. Here is the args that we used for most experiments (other args are using default options).
 
“--max_seq_length 512 --train_batch_size 16 --eval_batch_size 16 --gradient_accumulation_steps 2 --learning_rate 1e-5 --no_token_id”

We didn’t use large accumulation steps in our experiment. However the gradient is sensitive and you might want to search from 1e-4 to 1e-5 for your own experiment.

**Comment by katie-cathy-hunt** [comment]
First of all, thanks for the prompt response! 

Your paper stated that the batch size per GPU ranges from 128 to 32.

![image](https://user-images.githubusercontent.com/57769557/70757777-b2c45d00-1cf5-11ea-8ba9-68a6cbb471e6.png)

What made you guys change it to 16?
